content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst

```diff
@@ -133,507 +157,595 @@ types available by default:
-- The amount should be bigger than 100,000 ARS. 
-- A bank account type CBU must be related to the emisor, otherwise the invoice can’t 
-  be validated, having these errors messages for example:  
-  
-.. image:: argentina/argentina_edi_10.png
+.. image:: argentina/bank-account-relation-error.png
    :align: center
+   :alt: Bank account relation error.
 
-**Credit& Debit Notes:** When creating a Credit/Debit note related to a FCE document, it is 
-important take the next points into consideration:
+To set up the :guilabel:`Transmission Mode` go to settings and select one of either :guilabel:`SDC`
+or :guilabel:`ADC`.
+
+.. image:: argentina/transmission-mode.png
+   :align: center
+   :alt: Transmission Mode.
 
-- Use the Credit and Debit Note buttons, so the correct reference of the originator 
-  document passed to the note. 
+To change the :guilabel:`Transmission Mode` for a specific invoice, go to the
+:guilabel:`Other Info` tab and change it before confirming.
 
-.. image:: argentina/argentina_edi_11.png
+.. note::
+   Changing the :guilabel:`Transmission Mode` will not change the mode selected in
+   :guilabel:`Settings`.
+
+.. image:: argentina/transmission-mode-on-invoice.png
    :align: center
-   
-- The document letter should be the same than the originator document (either A or B).
-- The same currency as the source document must be used.  When using a secondary currency 
-  there is an exchange difference  if the currency rate is different between the emission day 
-  and the payment date, it’s possible to create a credit/debit note to decrease/increase the 
-  amount to pay in ARS. 
+   :alt: Transmission Mode on Invoice.
+
+When creating a :guilabel:`Credit/Debit` note related to a FCE document:
+
+- use the :guilabel:`Credit and Debit Note` buttons, so the correct reference of the originator
+  document passed to the note.
+- the document letter should be the same than the originator document (either A or B).
+- the same currency as the source document must be used. When using a secondary currency
+  there is an exchange difference  if the currency rate is different between the emission day
+  and the payment date, so it's possible to create a credit/debit note to decrease/increase the
+  amount to pay in ARS.
+
+.. image:: argentina/credit-debit-notes-button.png
+   :align: center
+   :alt: Credit & debit notes buttons.
 
 In the workflow we can have two scenarios:
 
-#. The FCE is rejected so the Credit Note should have the field “FCE, is Cancellation?” as True. 
-#. The Credit Note, is created with the negative amount to annulate the FCE document, 
-   in this case the field “FCE, is Cancellation?” must be empty (false). 
+#. The FCE is rejected so the :guilabel:`Credit Note` should have the field :guilabel:`FCE,
+   is Cancellation?` as *True*.
+#. The :guilabel:`Credit Note`, is created annul the FCE document, in this case the field
+   :guilabel:`FCE, is Cancellation?` must be *empty* (false).
```

Comment:
Grammar/formatting:
- lowercase first letter of first word in the list since it's a continuation of stem sentence before it; and
- use a semicolon here to split contrasting statements.

```suggestion
#. the FCE is rejected so the :guilabel:`Credit Note` should have the field :guilabel:`FCE,
   is Cancellation?` as *True*; or
#. the :guilabel:`Credit Note`, is created annul the FCE document, in this case the field
   :guilabel:`FCE, is Cancellation?` must be *empty* (false).
```